polaris-react/src/components/RadioButton/RadioButton.scss

```
@@ -165,7 +165,7 @@
   }
 
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  @include focus-ring($border-width: var(--p-border-width-2));
+  @include focus-ring($border-width: var(--p-border-width-0165));
```

```suggestion
  @include focus-ring($border-width: var(--p-border-width-050));
```

`--p-border-width-2` should be mapped to `--p-border-width-050` here.